MAINTAINERS

```diff
@@ -0,0 +1,152 @@
+request.
+
+2. Bug reports and questions should be posted to the GitHub project as well
+(use the "Issues" tab).
+
+3. The last entry ("THE REST") lists the overall maintainers (M:) and the
+members of the Linaro Security Working Group who provide reviews on a regular
+basis (R:).
+
+[1] https://www.kernel.org/doc/linux/MAINTAINERS
+[2] https://github.com/OP-TEE/optee_os
+
+----------
+
+ARM Foundation FVP
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-vexpress/
+
+ARM Juno
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-vexpress/
+
+Atmel ATSAMA5D2-XULT
+R:	Akshay Bhat <akshay.bhat@timesys.com> [@nodeax]
+S:	Maintained
+F:	core/arch/arm/plat-sam/
+
+HiSilicon D02
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+S:	core/arch/arm/plat-d02/
+
+HiSilicon HiKey (Kirin 620), HiKey960 (Kirin 960)
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-hikey/
+
+HiSilicon Poplar (Hi3798C V200)
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-poplar/
+
+Marvell Armada 70x0, Armada 80x0, Armada 3700
+R:	Kevin Peng <kevinp@marvell.com> [@kevin-peng]
+S:	Maintained
+F:	core/arch/arm/plat-marvell/
+
+MediaTek MT8173 EVB
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-mediatek/
+
+NXP LS1021A, LS1043A-RDB, LS1046A-RDB
+R:	Sumit Garg <sumit.garg@nxp.com> [@b49020]
+S:	Maintained
+F:	core/arch/arm/plat-ls/
+
+NXP LS1012A-RDB
+R:	Pankaj Gupta <pankaj.gupta@nxp.com> [@pangupta]
+S:	Maintained
+F:	core/arch/arm/plat-ls/
+
+NXP (Freescale) i.MX6 Quad SABRE Lite, i.MX6 Quad SABRE SD
+R:	Yan Yan <yan.yan@windriver.com> [@yanyan-wrs]
+R:	Feng Yu <Yu.Feng@windriver.com>
+S:	Maintained
+F:	core/arch/arm/plat-imx/
+
+NXP (Freescale) i.MX6 UltraLite EVK, i.MX6 ULL EVK, i.MX6 SX Sabreauto, i.MX7 Dual SabreSD
+R:	Peng Fan <peng.fan@nxp.com> [@MrVan]
+S:	Maintained
+F:	core/arch/arm/plat-imx/
+
+QEMU (32 and 64 bits)
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-vexpress/
+
+Raspberry Pi3
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-rpi3/
+
+Renesas RCAR
+R:	Volodymyr Babchuk <vlad.babchuk@gmail.com> [@lorc]
+S:	Maintained
+F:	core/arch/arm/plat-rcar/
+
+Rockchip RK322X
+R:	Rockchip <op-tee@rock-chips.com>
+S:	Maintained
+F:	core/arch/arm/plat-rockchip/
+
+Spreadtrum SC9860
+R:	Aijun Sun <aijun.sun@spreadtrum.com>
+S:	Maintained
+F:	core/arch/arm/plat-sprd/
+
+STMicroelectronics b2260-h410, b2120-h310/h410
+R:	Linaro <op-tee@linaro.org>
+S:	Maintained
+F:	core/arch/arm/plat-stm/
+
+Texas Instruments AM43xx, AM57xx, DRA7xx
+R:	Andrew F. Davis <afd@ti.com> [@glneo]
+S:	Maintained
+F:	core/arch/arm/plat-ti/
+
+Xilinx Zynq 7000 ZC702 Board
+R:	Yan Yan <yan.yan@windriver.com> [@yanyan-wrs]
+R:	Feng Yu <Yu.Feng@windriver.com>
+S:	Maintained
+F:	core/arch/arm/plat-zynq7k/
+
+Xilinx Zynq UltraScale+ MPSOC
+R:	Sören Brinkmann <soren.brinkmann@xilinx.com> [@sorenb-xlnx]
```

Comment:
According to Linked it looks like Sören might not do this any more.

Reply:
Aha, that might explain why we haven't heard from him in a while. We can ask him if Apple have any device he wants to put here ;-)

Reply:
;-)
OK I'll mark the platform as Orphan, then.